src/controllers/boardController.js

```diff
@@ -116,13 +114,7 @@ function updateGame(req, res) {
 	}).catch((err) => {
 		debug(`Error Updating Game: ${err.toString()}`);
 		//TODO: Add Proper Logging
-
-		//TODO: we shouldn't be switching behavior based on text matches. let's find a better way at some point.
-		if (err.toString().indexOf('No such') > -1) {
```

Comment:
this looks like there was a problem with the database throwing when it's not found -- did we verify that's no longer happening?

Reply:
I would only expect a  "No such" error, if it was a "No such table ..." or "No such column ..."
 error... which is certainly a 500-worthy situation rather than 404. I'd be very surprised if no result due to a "where" clause ever caused a "No such" error of any kind... hm.